### ISSUE TITLE
Record by default?

### ISSUE THREAD

**Issue by snypelife** [keithclark/gadebugger#58]
Just noticed there's a number of issues being submitted around not working or not seeing data that appear to be folks missing the record button.

I wonder if it would be useful to have the extension record by default upon activation of the tab?

**Comment by ArnaudLigny** [comment]
I agree with @snypelife: like the `Network` tab that record by default.
